virtwho/register.py

```diff
@@ -827,6 +853,19 @@ def associate(self, hypervisor, guest):
                     'Failed to find the associated hypervisor in guest page')
                 return False
 
+    def sca(self, sca='disable'):
+        """
+        Enable/disable simple content access.
+        :param sca: enable/disable.
+        :return: True or raise fail.
+        """
+        ret, output = self.ssh.runcmd(f'hammer simple-content-access {sca} '
+                                      f'--organization-id {self.org_id}')
+        if ret == 0 and '100%' in output:
+            logger.info(f'Succeeded to {sca} SCA for satellite')
+            return True
+        raise FailException(f'Failed to {sca} SCA for satellite')
+
 
 def request_get(url, auth, verify=False):
```

Comment:
would be great if we also have the docstring for request_get, request_put, request_delete

Reply:
Added docstring for each one, thanks.